### ISSUE TITLE
[BUG] a11y issue

### ISSUE THREAD

**Issue by jonas-hoebenreich** [formbricks/formbricks#2699]
### Issue Summary

The color contrast for the star input question is still insufficient:
https://color.a11y.com/ContrastPair/?bgcolor=ffffff&fgcolor=94A3B8 
![image](https://github.com/formbricks/formbricks/assets/64426524/44af9992-4487-4adb-b87e-c6ce077b613e)

see https://github.com/formbricks/formbricks/issues/2627

### Steps to Reproduce

1. Check contrast of star question

### Expected behavior

Website surveys comply with the Americans with Disabilities Act: https://www.ada.gov/resources/web-guidance/

### Screenshots

![image](https://github.com/formbricks/formbricks/assets/64426524/032b7c75-315f-4ee1-955c-141ddd0d5202)

### Environment

- [X] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS:
- Node:
- npm:
```

**Comment by mattinannt** [comment]
@jonas-hoebenreich Thanks for pointing this out. What is the a11y standard you're following? Is `WCAG AA` sufficient?

**Comment by jonas-hoebenreich** [comment]
Yes, `WCAG AA` is the required standard to be compliant with ADA
